go

```diff
@@ -34,6 +34,7 @@ def_command :update_gems, 'Update Ruby gems' do |gems = []|
 end
 
 def_command :test, 'Execute automated tests' do |args = []|
+  exec_cmd 'bundle exec rake test'
```

Comment:
Yeah, this ends up double-testing the schema: Once using the `rake` task, and again using your `rspec` test. The thing was, the `rspec` test was only able to test the `.about.yml` in _this_ repo, best I could tell. But the `rake` task can be included in any repo's `Rakefile` and validate that repo's `.about.yml`.